src/lab4.py

```diff
@@ -0,0 +1,43 @@
+class Lamp:
+    def __init__(self, type_="Нема типу", power=0, quantity=0, name="None", price=0, character="Нема характер"):
```

Comment:
Change the default value of `name` from "None" (string) to `None` (NoneType) for better clarity and consistency.